src/css/_utils.scss

```diff
@@ -1,16 +1,7 @@
-@mixin placeholder {
-  &::-webkit-input-placeholder,
-  &::-moz-placeholder,
-  &:-ms-input-placeholder {
-    @content;
-  }
-}
-
 @mixin user-select($spec...) {
   -webkit-touch-callout: $spec;
```

Comment:
@TrySound Isn't this obsolete? It could probably be removed IIRC.

Reply:
🤷‍♂️ Maybe